AntWars/ConfigurationPanel.cs

```diff
@@ -388,6 +382,31 @@ private void btn_gameConfigNew_Click(object sender, EventArgs e)
         {
             configLoader.newGame();
             configLoadedOrNewCreatedGame();
+            getStandardGameValues();
+        }
+
+        private void getStandardGameValues()
```

Comment:
``configLoader.newGame();`` in der methode gleich die standart werte setzten dann in ``ConfigurationLoader.cs`` nach dem newGame() das aufrufen ``loadGame(configLoader.get().Game);`` 

ja es verhindert unnötigen code